Assets/MixedRealityToolkit/Extensions/GameObjectExtensions.cs

```diff
@@ -158,5 +158,50 @@ public static void DestroyGameObject(GameObject gameObject, float t = 0.0f)
         {
             UnityObjectExtensions.DestroyObject(gameObject, t);
         }
+
+        /// <summary>
+        /// Checks if any MonoBehaviour on the given GameObject is using the RequireComponentAttribute requiring type T
+        /// </summary>
+        /// <typeparam name="T">The potentially required component</typeparam>
+        /// <param name="gameObject">the GameObject requiring the component</param>
+        /// <param name="requiringTypes">A list of types that do require the component in question</param>
+        /// <returns></returns>
```

Comment:
```suggestion
```

Reply:
This line is triggering a validation failure (we have some code that checks we don't have empty doccomments)